server/controllers/content.js

```diff
@@ -12,17 +12,20 @@ import {
 import {PromoListFactory} from '../model/promo-list';
 import {PaginationFactory} from '../model/pagination';
 import {getInfoPage} from '../../common/services/prismic/info-page';
+import {getCollectionOpeningTimes} from '../../common/services/prismic/opening-times';
 
-export const renderOpeningTimes = (ctx, next) => {
+export const renderOpeningTimes = async(ctx, next) => {
```

Comment:
Does this(/should this) break the linter? I thought a space would be required after `async`